js/members.js

```diff
@@ -5,9 +5,10 @@ $(document).ready(function() {
         {
             url: "/data/members.json",
             success: function(result) {
-                loadFourthYears(result[2018]);
-                loadThirdYears(result[2019]);
-                loadSecondYears(result[2020]);
+                loadFirstAlumni(result[2018]);
```

Comment:
rename to ```loadAllAlumni()```, which in turn calls a function ```load2018Alumni(result[2018])```

Reply:
done